examples/advanced/quantum_volume.py

```diff
@@ -104,6 +104,41 @@ def compute_heavy_set(circuit: cirq.Circuit) -> List[int]:
     ]
 
 
+def sample_heavy_set(circuit: cirq.Circuit,
+                     heavy_set: List[int],
+                     *,
+                     sampler: cirq.Sampler = cirq.Simulator()) -> float:
+    """Run a sampler over the given circuit and compute the percentage of its
+       outputs that are in the heavy set.
+
+    Args:
+        circuit: The circuit to sample.
+        heabv_set: The previously-computed heavy set for the given circuit.
+        sampler: The sampler to run on the given circuit.
+
+    Returns:
+        A probability percentage, from 0 to 1, representing how many of the
+        output bit-strings were in the heaby set.
+
+    """
+    # Add measure gates to the end of the circpuit.
+    circuit_copy = circuit.copy()  # So we don't mutate the original circuit.
+    circuit_copy.append([cirq.measure(*sorted(circuit_copy.all_qubits()))])
+
+    # Run the sampler to compare each output against the Heavy Set.
+    repetitions = 1000
+    measurements = sampler.run(program=circuit_copy, repetitions=repetitions)
+
+    # Compute the number of outputs that are in the heavy set.
+    num_in_heavy_set = 0
+    for _, m in enumerate(measurements.data.iterrows()):
```

Comment:
You're enumerating to add the index, then ignoring the index. Just iterate without enumerate.

Reply:
Woops, nice catch. Done.